CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+Change Log
+==========
+
+Version History
+---------------
+
+0.1.1
```

Comment:
Well the original idea was to have those as links, in case the version history got long, to make navigation easier. However, I couldn't get the links to work, or at least they didn't appear to work in the Atom editor. I can remove the section if you want.